maintainers-circle/README.md

```diff
@@ -60,6 +60,45 @@ Join our <a href="https://lists.cncf.io/g/cncf-tag-contributor-strategy">mailing
 
 ## What
 
+### Community Management
+
+June 28th, 2023 @ 10:00am PT / 5:00pm GMT / [Your timezone here](https://time.is/compare/1000AM_28_June_2023_in_PT)
```

Comment:
```suggestion
June 28th, 2023 @ 10:00am PT / 17:00 UTC / [Your timezone here](https://time.is/compare/1000AM_28_June_2023_in_PT)
```

I changed this to UTC from GMT, since I don't think anyone is on GMT right now. Here in the UK, we're on BST (British Summer Time) because of daylight saving time.